api/metrics/community_test.py

```diff
@@ -210,4 +210,47 @@ def test_belonging():
         { "area_number": 1, "value": 67.1 / 100 }
     ], "Should have one result for 2018."
     
-    assert metric.belonging(year=2015, segment="all") == [], "Should have no results for 2015."
\ No newline at end of file
+    assert metric.belonging(year=2015, segment="all") == [], "Should have no results for 2015."
+    
+def test_disabilities():
+    disabilities_table = [
+        {
+            "area_number": 1,
+            "period_end_year": 2018,
+            "segment": "all",
+            "value": 13.37
+        },
+        {
+            "area_number": 2,
+            "period_end_year": 2016,
+            "segment": "all",
+            "value": 11.02
+        },
+        {
+            "area_number": 3,
+            "period_end_year": 2016,
+            "segment": "all",
+            "value": 8.46
+        },
```

Comment:
Good test! I would add one more row with a segment that is not "all" to show that the segment filter works.